packages/core/src/codewhispererChat/commands/registerCommands.ts

```diff
@@ -37,73 +36,13 @@ export const focusAmazonQPanelKeybinding = Commands.declare('_aws.amazonq.focusC
     await focusAmazonQPanel.execute(placeholder, 'keybinding')
 })
 
-const getCommandTriggerType = (data: any): EditorContextCommandTriggerType => {
-    // data is undefined when commands triggered from keybinding or command palette. Currently no
-    // way to differentiate keybinding and command palette, so both interactions are recorded as keybinding
-    return data === undefined ? 'keybinding' : 'contextMenu'
-}
-
-export function registerCommands(controllerPublishers: ChatControllerMessagePublishers) {
```

Comment:
Just to verify, these still register properly on non-hybrid chat right?

Reply:
yeah this is only broken for hybrid chat, everything works fine for the regular main branch